src/devices/namron.ts

```diff
@@ -1617,18 +1617,25 @@ export const definitions: DefinitionWithExtend[] = [
         ],
     },
     {
-        zigbeeModel: ["4512792"],
-        model: "4512792",
-        vendor: "Namron",
-        description: "Simplify 1-2p relay (Zigbee / BT)",
+        zigbeeModel: ["4512791"],
+        model: "4512791",
+        vendor: "Namron AS",
```

Comment:
```suggestion
        vendor: "Namron",
```

(should be the same as all other `vendor` in this file)